From dbb20c1e06ecf96f6a13f2e2cb6bba050da7c23a Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Fri, 13 Feb 2009 16:17:02 +0530
Subject: [PATCH 286/364] defconfig updated for video

- DMA size made 12
- DSS2 and OMAP_FB made mutual exclusive
- Made nmber of fbs to 1
---
 drivers/video/omap/Kconfig  |    3 ++-
 drivers/video/omap2/Kconfig |    7 ++++---
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/drivers/video/omap/Kconfig b/drivers/video/omap/Kconfig
index 463e907..5d2c897 100644
--- a/drivers/video/omap/Kconfig
+++ b/drivers/video/omap/Kconfig
@@ -5,6 +5,7 @@ config FB_OMAP
 	select FB_CFB_FILLRECT
 	select FB_CFB_COPYAREA
 	select FB_CFB_IMAGEBLIT
+	select FB_BACKLIGHT
 	default y if (OMAP2_DSS = "n")
 	help
           Frame buffer driver for OMAP based boards.
@@ -84,7 +85,7 @@ config FB_OMAP_CONSISTENT_DMA_SIZE
 	int "Consistent DMA memory size (MB)"
 	depends on FB && ARCH_OMAP
 	range 1 14
-	default 2
+	default 12
 	help
 	  Increase the DMA consistent memory size according to your video
 	  memory needs, for example if you want to use multiple planes.
diff --git a/drivers/video/omap2/Kconfig b/drivers/video/omap2/Kconfig
index 7753000..e8374f9 100644
--- a/drivers/video/omap2/Kconfig
+++ b/drivers/video/omap2/Kconfig
@@ -1,11 +1,12 @@
 config FB_OMAP2
         tristate "OMAP2/3 frame buffer support (EXPERIMENTAL)"
-        depends on FB && OMAP2_DSS
+        depends on FB && ARCH_OMAP
 
         select FB_CFB_FILLRECT
         select FB_CFB_COPYAREA
         select FB_CFB_IMAGEBLIT
-	default OMAP2_DSS if (FB_OMAP = "n")
+	select FB_BACKLIGHT
+	select OMAP2_DSS
         help
           Frame buffer driver for OMAP2/3 based boards.
 
@@ -29,7 +30,7 @@ config FB_OMAP2_FORCE_AUTO_UPDATE
 config FB_OMAP2_NUM_FBS
 	int "Number of framebuffers"
 	range 1 3
-	default 3
+	default 1
 	depends on FB_OMAP2
 	help
 	  Select the number of framebuffers created. OMAP2/3 has 3 overlays
-- 
1.5.6

